src/index.html

```diff
@@ -1,17 +1,184 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="group-container">
+        <fieldset>
+          <legend>Personal information</legend>
+          <div class="container">
+            <label for="surname">Surname:</label>
+            <input
+              type="text"
+              id="surname"
+              required
+            >
+          </div>
+          <div class="container">
+            <label for="name">Name:
```

Comment:
`<label for="name">Name:</label>`

Reply:
didn't fix it, when you wrap the inputs you don't need for, use for without wrap inputs, or delete for and add spaces
```
<label for="name">
   Name:
   <input
```